src/easylink/implementation_metadata.yaml

```diff
@@ -5,6 +5,18 @@ step_1_python_pandas:
   script_cmd: python /dummy_step.py
   outputs:
     step_1_main_output: result.parquet
+step_1a_python_pandas:
```

Comment:
added as "proof" that parallel step works with Hierarchical node

Reply:
This is confusing to me. Isn't this implementation_metadata intended to be a sort of record of what exactly EasyLink supports? This change seems more like a test thing.

Reply:
Yeah, I guess in a sense adding it to the "true" pipeline schema is a bit of a lazy way to avoid mocking the implementation metadata and pipeline schema for the step unit tests and e2e tests, which eventually we'll have to do anyway.